packages/terra-docs/src/terra-dev-site/tool/DocumentationComponents/ChangeLog.z.tool.mdx

```diff
@@ -0,0 +1,6 @@
+import { Badge } from '@cerner/terra-docs/package.json?dev-site-package';
+import ChangeLog from '@cerner/terra-docs/CHANGELOG.md';
+
+<Badge />
+
+<ChangeLog />
```

Comment:
Should we start writing the changelogs here??

Reply:
We'd have to move the changelog from root and make it an MDX doc... I think i prefer keeping it where it is and keeping it as a MD doc. (github renders md better than mdx)